polar2grid/writers/hdf5.py

```diff
@@ -60,7 +60,14 @@
 LOG = logging.getLogger(__name__)
 
 # reader_name -> filename
-DEFAULT_OUTPUT_FILENAMES = {None: "{platform_name}_{sensor}_{start_time:%Y%m%d_%H%M%S}.h5"}
+DEFAULT_OUTPUT_FILENAMES = {
+    None: "{platform_name!u}_{sensor!u}_{p2g_name}_{start_time:%Y%m%d_%H%M%S}_{area.area_id}.tif",
+    "abi_l1b": "{platform_name!u}_{sensor!u}_{observation_type}{scene_abbr}_"
+    "{p2g_name}_{start_time:%Y%m%d_%H%M%S}_{area.area_id}.tif",
+    "viirs_sdr": "{platform_name!l}_{sensor!l}_{p2g_name}_{start_time:%Y%m%d_%H%M%S}_{area.area_id}.tif",
+    "mirs": "{platform_name!l}_{sensor!l}_{p2g_name}_{start_time:%Y%m%d_%H%M%S}_{area.area_id}.tif",
+    "virr_l1b": "{platform_name!l}_{sensor!l}_{p2g_name}_{start_time:%Y%m%d_%H%M%S}_{area.area_id}.tif",
```

Comment:
Change `.tif` to `.h5`

Reply:
Oh, actually you should remove `p2g_name` too and probably `area id`? Not sure what the old P2G produced.

Reply:
I reverted this back to the single line default.  Looking at old test output on bumi, hdf5 data all have the same default filename convention.

Reply:
Yeah, that's fine. This writer didn't exist for Geo2Grid which is really the only reason there are different naming patterns in these dictionaries anyway.